volontulo/tests/views/test_offers.py

```diff
@@ -475,3 +475,165 @@ def test_for_correct_slug(self):
         self.assertIn('offer', response.context)
         self.assertIn('volunteers', response.context)
         self.assertEqual(len(response.context['volunteers']), 5)
+
+
+class TestOffersJoin(TestCase):
+    u"""Class responsible for testing offer's join page."""
```

Comment:
Is there any reason for this constraint? We right now have test divided for modules reagarding object, that is involved in and separate classes for every view.